consensus/obcpbft/pbft-core.go

```diff
@@ -252,6 +255,11 @@ func newPbftCore(id uint64, config *viper.Viper, consumer innerStack) *pbftCore
 	} else {
 		logger.Info("PBFT null requests disabled")
 	}
+	if instance.viewChangePeriod > 0 {
```

Comment:
I think this would be superior, though I'm fine if you'd like to do this is a subsequent PR.